bzt/modules/aggregator.py

```diff
@@ -165,6 +167,8 @@ def recalculate(self):
 
         :return:
         """
+        self._compact_times()
```

Comment:
We don't need to compact times for each KPIset. We only need to do it for cumulative KPIsets.